### PR TITLE
TA-2380: Update screenshots for API Users

### PR DESCRIPTION
**Pull request by mhs-trackunit**

### CODE REVIEW

**Review by ksj-trackunit — APPROVED** [review]

File: api-docs/access-token.md [hunk 1]
```diff
@@ -20,17 +20,24 @@ Visit the Manager application to create an API User and obtain OAuth 2.0 credent
 > 
 > API Users will act as the admin user. Only the admin user can access the "API Access"-page to create API Users and obtain credentials.
 
-Find the "API Access" page in the upper right corner of the Manager application.
+Find the "API Access" page under Settings → Administration.
```
Comment (bkaminski-tu) [inline api-docs/access-token.md:23]: Settings > Administration? It's Administration > API Access, right? 🤔

File: api-docs/access-token.md [hunk 2]
```diff
@@ -20,17 +20,24 @@ Visit the Manager application to create an API User and obtain OAuth 2.0 credent
...
 
-![](https://files.readme.io/af2792b-d4792b9-a8de0f7-Skrmbillede_2020-04-15_kl._12.33.13.png "d4792b9-a8de0f7-Skrmbillede_2020-04-15_kl._12.33.13.png")
+![Screenshot of snippet showing API Access under Administration Settings](https://cdn.statically.io/gh/trackunit/developer-hub/master/api-docs/api-access-admin-settings.png "API Access settings")
```
Comment (bkaminski-tu) [inline api-docs/access-token.md:25]: Should these links work already now? They don't work when I copy paste them into a browser 🤔

File: api-docs/access-token.md [hunk 3]
```diff
@@ -20,17 +20,24 @@ Visit the Manager application to create an API User and obtain OAuth 2.0 credent
...
 
-![](https://files.readme.io/4b969c6-f2791eb-a04e37e-Skrmbillede_2020-04-14_kl._16.39.47.png "f2791eb-a04e37e-Skrmbillede_2020-04-14_kl._16.39.47.png")
+![Screenshot of dialog with name and description for API User](https://cdn.statically.io/gh/trackunit/developer-hub/master/api-docs/api-access-create-api-user-dialog.png "Dialog of API User creation")
+
+Capture the username and password of created user along with the "Client ID" and "Client Secret".
+> 📘 Password
+>
+> Remember to save the password. The password will only be visible this **_one time_**.
+
+![Screenshot showing credentials needed to authenticate](https://cdn.statically.io/gh/trackunit/developer-hub/master/api-docs/api-access-created-user.png "Created API User")
```
Comment (bkaminski-tu) [inline api-docs/access-token.md:40]: Would it make sense to capture a screen shot with all this data masked? 🤔

**Review by bkaminski-tu — APPROVED** [review]

File: api-docs/access-token.md (hunk 2 above)
Comment (mhs-trackunit) [reply api-docs/access-token.md:25]: No, they'll work when merged to master

File: api-docs/access-token.md (hunk 3 above)
Comment (mhs-trackunit) [reply api-docs/access-token.md:40]: I modified the data so it is useless 😄

Comment (bkaminski-tu) [reply api-docs/access-token.md:40]: I was thinking more of a good practice not to show anything like that :D

**Pull request merged**